server/controllers/temporary-accommodation/manage/bedspacesController.ts

```diff
@@ -109,13 +109,15 @@ export default class BedspacesController {
       const { premisesId, roomId } = req.params
 
       const premises = await this.premisesService.getPremises(callConfig, premisesId)
+      const premisesCharacteristics = premises.characteristics.map(item => item.name).sort()
```

Comment:
My preference would be to use `localeCompare` for anything being presented to users - so something like `.sort(a.localeCompare(b))`

Reply:
In practice it doesn't make too much difference, but it's saying, sort this taking into account non-obvious language subtleties, rather than the order characters appear in ASCII

Reply:
thanks!